.github/actions/setup_flutter/action.yml

```
@@ -7,14 +7,68 @@ outputs:
+      shell: bash
+      working-directory: packages/app
+      run: |
+        touch .env
+        echo "SUPABASE_URL=${{ secrets.SUPABASE_URL }}" >> .env
+        echo "SUPABASE_API_KEY=${{ secrets.SUPABASE_API_KEY }}" >> .env
+        echo "BANNER_ID_IOS=${{ secrets.BANNER_ID_IOS }}" >> .env
+        echo "BANNER_ID_ANDROID=${{ secrets.BANNER_ID_ANDROID }}" >> .env
+        echo "INTERSTITIAL_ID_IOS=${{ secrets.INTERSTITIAL_ID_IOS }}" >> .env
+        echo "INTERSTITIAL_ID_ANDROID=${{ secrets.INTERSTITIAL_ID_ANDROID }}" >> .env
+        echo "REWARD_ID_ANDROID=${{ secrets.REWARD_ID_ANDROID }}" >> .env
+        echo "REWARD_ID_IOS=${{ secrets.REWARD_ID_IOS }}" >> .en
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)